src/api/telemetry/TelemetryAPI.js

```diff
@@ -324,6 +326,17 @@ define([
             return Promise.reject('No provider found');
```

Comment:
I want to change how we handle this case. Instead of rejecting the promise (which forces all of our views to add a bunch of boilerplate to handle errors) let's move the notification you added at line 322 up to here, as well as logging a console error, and resolve with an empty array.